wremnants/muon_efficiencies_binned_vqt_real_old.py

```diff
@@ -0,0 +1,267 @@
+import ROOT
```

Comment:
Is it expected for this file to still be used? If not I'm not sure I'd commit it.

Reply:
Actually not, but I kept it to do some debugging, because I expect it to give the same results as the one with 3D hists, but it doesn't, so I wanted to keep it for debugging, but I guess it can be removed